springboot-jersey-app/styling/src/main/java/com/wfsample/styling/StylingController.java

```diff
@@ -0,0 +1,71 @@
+package com.wfsample.styling;
+
+import com.wfsample.common.BeachShirtsUtils;
+import com.wfsample.common.dto.PackedShirtsDTO;
+import com.wfsample.common.dto.ShirtDTO;
+import com.wfsample.common.dto.ShirtStyleDTO;
+import com.wfsample.common.dto.DeliveryStatusDTO;
+import com.wfsample.service.DeliveryApi;
+import com.wfsample.service.StylingApi;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.ThreadLocalRandom;
+
+import javax.ws.rs.core.Response;
+
+import static java.util.stream.Collectors.toList;
+
+/**
+ * Driver for styling service which manages different styles of shirts and takes orders for a shirts
+ * of a given style.
+ *
+ * @author Srujan Narkedamalli (snarkedamall@wavefront.com).
+ */
+public class StylingController implements StylingApi {
+  private final DeliveryApi deliveryApi;
+  private List<ShirtStyleDTO> shirtStyleDTOS;
+
+  StylingController() {
+    String deliveryUrl = "http://localhost:50052";
+    this.deliveryApi = BeachShirtsUtils.createProxyClient(deliveryUrl, DeliveryApi.class);
+    shirtStyleDTOS = new ArrayList<>();
+    ShirtStyleDTO dto = new ShirtStyleDTO();
+    dto.setName("style1");
+    dto.setImageUrl("style1Image");
+    ShirtStyleDTO dto2 = new ShirtStyleDTO();
+    dto2.setName("style2");
+    dto2.setImageUrl("style2Image");
+    shirtStyleDTOS.add(dto);
+    shirtStyleDTOS.add(dto2);
+  }
+
+  public List<ShirtStyleDTO> getAllStyles() {
+    return this.shirtStyleDTOS;
+  }
+
+  public Response makeShirts(String id, int quantity) {
+    /*
+     * TODO: Try to report the value of quantity using WavefrontHistogram.
```

Comment:
Add this to the multi-line TODO

Viewing the quantity requested by various clients as a minute distribution and then applying statistical functions (median, mean, min, max, p95, p99 etc.) on that data is really useful to understand the user trend.